commands/anonSuggest.js

```diff
@@ -0,0 +1,39 @@
+const {GuildChannel} = require('discord.js');
+
+module.exports = {
+	name: 'anonsuggest',
+	description: 'Posts an anonymous Suggestion',
+	usage: 'anonsuggest',
+	helpMsg: true,
+	execute(message, args) {
+		const {modBot} = message.client.config.channelIDs;
+		const suggestion = args.join(' ');
+
+		const lastChar = args[0].slice(args[0].length - 1, args[0].length);
+		let number;
+
+		if (!isNaN(lastChar)) {
+			number = args[0];
+		}else {
+			number = args[0].slice(0, args[0].length - 1);
+		}
+
+		if (isNaN(number)) {
+			message.author.send('Please put a number at the start');
+			if (message.channel instanceof GuildChannel) {
```

Comment:
Don't think you need this here as L35 should pick this up and delete it

Reply:
Now it's needed due to return.